mne/io/eeglab/eeglab.py

```diff
@@ -609,9 +609,10 @@ def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
     if len(events) < len(types):
         warn("Some event codes could not be mapped to integers. Use the "
              "`event_id` parameter to map such events to integers manually.")
-    if len(events) < 1:
-        warn("No events found, consider adding an `event_id`. As is, the "
-             "trigger channel will consist entirely of zeros.")
+    if len(events) < 1 and len(types) > 0:
```

Comment:
Sure logging info is fine but not warning

On Sep 1, 2016 1:56 PM, "jona-sassenhagen" <notifications@github.com> wrote:

> In mne/io/eeglab/eeglab.py
> <https://github.com/mne-tools/mne-python/pull/3557#discussion_r77222339>:
>
> > @@ -609,9 +609,10 @@ def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
> >      if len(events) < len(types):
> >          warn("Some event codes could not be mapped to integers. Use the "
> >               "`event_id` parameter to map such events to integers manually.")
> > -    if len(events) < 1:
> > -        warn("No events found, consider adding an `event_id`. As is, the "
> > -             "trigger channel will consist entirely of zeros.")
> > +    if len(events) < 1 and len(types) > 0:
>
> Would you say there should be NO warning if no events exist in the
> original file? I think that should still be logged somehow ("No events
> detected, returning empty events")
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3557/files/03dff95ee959427266df15a6f0187a31e2d0075d#r77222339>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACQZXogzhfTh8D8ePVh9rOMQ920t4fNgks5qlxG3gaJpZM4Jy8P3>
> .
>